### ISSUE TITLE
[GUI] Constant definitions for verify action get lost after model reload

### ISSUE THREAD

**Issue by merkste** [prismmodelchecker/prism#206]
After reloading a model, constants defined in a previous run get lost. Interestingly, the constants defined in a previous _experiment_ are preserved. After running an experiment, the constant definitions for verify are restored (or taken over from the experiment).

**Comment by davexparker** [comment]
Yes, good spot. This changed recently, in 95a6c582bd70d5881762fd38bd64cefe953c02e7. That tries to keep more functionality in the `prism.Prism` API and avoid duplicating it in the GUI. But this "reload model and remember constants" breaks. Worth thinking about.
